fix_checkpoint_history.py

```
@@ -0,0 +1,80 @@
+    
+    with open(checkpoint_file, 'w') as f:
+        json.dump(checkpoint_data, f, indent=2)
+    
+    print(f"\nUpdated checkpoint to resume from location {estimated_locations}")
+    
+    # Create/update history file
+    history_file = Path("data/openaq/checkpoints/checkpoint_history.json")
+    history = {}
+    
+    if history_file.exists():
+        with open(history_file, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)